### ISSUE TITLE
Label flush incorrect for reversed scales

### ISSUE THREAD

**Issue by domoritz** [vega/vega#1278]
```json
{
  "$schema": "https://vega.github.io/schema/vega/v3.json",
  "width": 200,
  "scales": [
    {
      "name": "xscale",
      "domain": [0, 1000000],
      "nice": true,
      "range": "width",
      "reverse": true
    }
  ],
  "axes": [
    {
      "orient": "bottom",
      "scale": "xscale",
      "labelFlush": true,
      "labelOverlap": true
    }
  ]
}
```

![image](https://user-images.githubusercontent.com/589034/40260928-6ba14598-5ab2-11e8-8681-fceb187f9724.png)

**Comment by jheer** [comment]
Thanks! Will fix.

**Comment by jheer** [comment]
Fixed in vega-parser 3.1.0: https://github.com/vega/vega-parser/releases/tag/v3.1.0
